Baekjoon/10866.py

```diff
@@ -0,0 +1,54 @@
+'''
+문제 링크:https://www.acmicpc.net/problem/10866
+카테고리:덱
+문제해설:덱(Deque)를 구현한 다음, 입력으로 주어지는 명령을 처리
+'''
+from collections import deque
+import sys
+
+n=int(input())
+input=sys.stdin.readline
+
+d=deque([])
+
+for i in range(n):
+  commend=input().split()
+
```

Comment:
```suggestion
for i in range(n):
  commends = input().split()
  commend = commends[0]

```
commend가 split에 의해서 배열로 받으니 원하는 커멘트를 얻기 위해서 `[0]`에 접근해야 합니다

Reply:
이부분 때문에 출력값이 안나왔던거군용